week1/lesson-plan.md

```diff
@@ -54,6 +54,30 @@ If you find anything that could be improved then please create a pull request! W
     - [Live coding](https://github.com/HackYourFuture-CPH/node.js/tree/main/week1/teacher-live-coding)
     - [Exercise](https://github.com/HackYourFuture-CPH/node.js/tree/main/week1/exercise-template)
 
+## Warmup exercises 
+
+### Create your hello world server 
+* Create a file called server.js
+* Create a server in this file.
+* Listen on port 3000.
+* When the user calls "/" return the text "hello world"
```

Comment:
It's a little semantic, but I think it would be good to teach students to use the language of HTTP so they start to understand that it's not JavaScript only anymore.

I would say you "call a function" but I would say you "make a request to an endpoint". Request/Response is a key concept in server development and I think it would be good to equip students with that vocabulary.

Specially since in the express handler they'll have access to exactly those two variables, req/res.